src/index.d.ts

```diff
@@ -217,6 +217,10 @@ export interface CommonMarkerProps {
   markerEnd?: string;
 }
 
+export interface LayoutProps {
```

Comment:
There are also a few other props missing, as `style`. I wonder if we should use a more generic name instead, as maybe `NativeProps` or `ViewProps` to allow us to add the other missing props here too.

Reply:
`NativeProps` sounds good. I'll rename `LayoutProps`